ansible/roles/grafana/files/grafana-default.ini

```diff
@@ -0,0 +1,816 @@
+##################### Grafana Configuration Example #####################
```

Comment:
Can we eliminate any "reference-only" files like this one `grafana-default.ini` - it seems to be unused by any actual task?

Reply:
Yes, I added `grafana-default.ini` to adopt the style used in the existing `influxdb` roles that had the default config file under `influxdb/files` directory. I'll go-ahead and remove `grafana-default.ini` & also `influxdb/files/influxdb-1.8.3-defaults.conf` - both not used in any tasks.

Reply:
done